cli/auth.go

```diff
@@ -493,7 +494,7 @@ func (a *authFlow) loginAsUser(ctx context.Context) (*token, error) {
 
 	err = a.directUser(deviceCode)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("Unable to open the browser to complete the login flow due to \"%w\". You can use the --disable-browser-open flag to skip this behavior.", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to open the browser to complete the login flow due to %w. You can use the --%s flag to skip this behavior.", err, loginFlagDisableBrowser)
```
Can't uppercase errors in go due to linting (they'll get uppercased before being passed to the user. Some other small edits too.